lib/at_shell/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+if(CONFIG_AT_SHELL)
```

Comment:
Is this flagging needed? We have flags at higher level protecting the compilation of this library.

Reply:
You're right, this is not needed, will remove